src/js/profile/mobile/widget/Calendar.js

```diff
@@ -0,0 +1,293 @@
+/*global define, ns */
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd
+ *
+ * Licensed under the Flora License, Version 1.1 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://floralicense.org/license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/*
+ * @example
+ * <div class="ui-calendar"></div>
+ *
+ * @since 1.2
+ * @class ns.widget.mobile.Calendar
+ * @extends ns.widget.BaseWidget
+ * @author Dohyung Lim <delight.lim@samsung.com>
+ */
+(function (document, ns) {
+	"use strict";
+	//>>excludeStart("tauBuildExclude", pragmas.tauBuildExclude);
+	define(
+		[
+			"../../../core/engine",
+			"../../../core/util/DOM/attributes",
+			"../../../core/event",
+			"../widget"
+		],
+		function () {
+			//>>excludeEnd("tauBuildExclude");
+			var utilsObject = ns.util.object,
+				engine = ns.engine,
+				events = ns.event,
+				days = ["MON", "TUE", "WED", "THU", "FRI", "SAT", "SUN"],
+				fullNameMonth = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"],
+				today = new Date(),
+				todayYear = today.getFullYear(),
+				todayMonth = today.getMonth() + 1,
+
+				Calendar = function () {
+					this.options = utilsObject.merge({}, defaultOptions);
+				},
+
+				defaultOptions = {
+					pastSelection: false
+				},
+
+				BaseWidget = ns.widget.BaseWidget,
+				prototype = new BaseWidget();
+
+			Calendar.prototype = prototype;
+
+			/**
+			* Init widget
+			* @method _init
+			* @param {HTMLElement} element
+			* @return {HTMLElement} Returns built element
+			* @member ns.widget.mobile.Calendar
+			* @protected
+			*/
+			prototype._init = function (element) {
+				var self = this;
+
+				if (!element.getAttribute("value")) {
+					element.setAttribute("value", self.options.value);
+				}
+
+				self._buildCalendar();
+
+				return element;
+			};
+
+			/**
+			* Draw a calendar on the table
+			* @method _buildCalendar
+			* @member ns.widget.mobile.Calendar
+			* @protected
+			*/
+			prototype._buildCalendar = function () {
+				var firstDate = new Date(todayYear, todayMonth - 1, 0),
+					lastDate = new Date(todayYear, todayMonth, 0),
+					day = firstDate.getDay(),
+					prevLastDate = lastDate.getDate(),
+					week = Math.ceil(prevLastDate / 7) + 1,
+					leftDays = 7,
+					setDays = 1,
+					nextMonthDate = 1,
+					cell = null,
+					calendar = document.getElementById("calendar"),
```

Comment:
change this to:
`calendar = this.element.querySelector(".calendarView")`

Reply:
done.